src/modules/objectsearch/ObjectSearchComponent.js

```diff
@@ -197,6 +198,21 @@ export class ObjectSearchComponent extends React.Component {
                                         />}
                                   </a>}
                               </td>
+                              <td>
+                                <a
```

Comment:
This is a misuse of the `a` element. We should use the `button` element and style it as a link.